nano/node/network.cpp

```diff
@@ -132,6 +135,7 @@ void nano::network::stop ()
 		{
 			thread.join ();
 		}
+		cleanup_timer.cancel ();
```

Comment:
Does this operation to be queued in a strand?
Also think we should use the overload with error_code to swallow exceptions.

Reply:
The documentation says it kills any asynchronous operation and the error_code variant is deprecated.
https://www.boost.org/doc/libs/1_71_0/doc/html/boost_asio/reference/basic_waitable_timer/cancel.html